File: week1/김고은/11650_좌표정렬하기/11650.js
let fs = require('fs');
let filePath = process.platform === 'linux' ? '/dev/stdin': __dirname + '/input.txt';
let input = fs.readFileSync(filePath).toString().trim().split('\n');

const N = +input.splice(0,1)[0];

let coordinate_arr = Array.from({length: N})

coordinate_arr.forEach((_, idx) => {
    coordinate_arr[idx] = input[idx].split(' ').map(Number)
})

coordinate_arr.sort((a, b) => {
    if(a[0] !== b[0])return a[0] - b[0]
    else return a[1] - b[1] 
})

coordinate_arr.map((e) => {
    console.log(e.join(" "))
})

File: week1/장수빈/11650.py
# 좌표 정렬하기

import sys

n = int(sys.stdin.readline()) # n: 좌표의 개수
arr = []

for _ in range(n):
    x, y = map(int, (sys.stdin.readline().split()))
    arr.append([x, y])

# 먼저 x를 기준으로 정렬, x가 같을 경우 y를 기준으로 정렬
arr.sort(key=lambda i: (i[0], i[1]))

for nums in arr:
    print(' '.join(map(str, nums)))